src/App.tsx

```diff
@@ -1,18 +1,32 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './App.css';
+
+import { Pagination } from './components/Pagination';
+
 import { getNumbers } from './utils';
 
-// eslint-disable-next-line @typescript-eslint/no-unused-vars
-const items = getNumbers(1, 42)
+const itemsList = getNumbers(1, 42)
   .map(n => `Item ${n}`);
 
 export const App: React.FC = () => {
+  const [perPage, setPerPage] = useState(5);
+  const [currentPage, setCurrentPage] = useState(1);
+  const [items] = useState(itemsList);
```

Comment:
why do you need to store a list of consecutive numbers in the state and pass it to the `Pagination `component?
you can store just a number and create an list where it needed(in the `Pagination`)